API.md

```diff
@@ -95,10 +115,18 @@
         }
 ]}
 ```
+Note the message response contains @msg_from , @msg_to and @ru . These hold values that the secure message api has resolved from the party service or the user authentication service. They are not guaranteed to be populated if the service is not available or if the message was sent to 'GROUP'
 
-## `POST /message/send`
+## `POST /message/send`  or '/v2/messages'
 
-#### Example JSON DATA for post
+The messages post endpoint stores a secure message . If the recipient is a respondent it will also send an email via Notify.Gov. Then inform the case service that a message has been sent 
+
+Note, the message post must have a Content-Type header of `application/json` , else it will return an error.
+Note, V2 uses messages , V1 uses message (singular)
+
+When a message is posted then typically no msg_id is supplied . If a msg_id is supplied then secure message will assume that the message was previously stored as a draft . It will then validate that it was a draft , send it and delete the draft. If it was not a draft and msg_id was set then it will return an error. Also there is an optional method to detect changes if multiple users edit a draft and then post. (See Drafts Put for description of Etags)
```

Comment:
The See Drafts Put bit, putting a link to it would be useful - https://gist.github.com/asabaylus/3071099